resources/routing_rule.rb

```diff
@@ -0,0 +1,48 @@
+property :description, ::String, default: ''.freeze
+property :mode, ::String, equal_to: %w[BLOCK ALLOW], default: 'BLOCK'.freeze
+property :matchers, Array, default: lazy { [] }, coerce: proc { |r| r.sort }
+property :api_client, ::Nexus3::Api, identity: true, default: lazy { ::Nexus3::Api.default(node) }
+
+load_current_value do |desired|
+  begin
+    config = api_client.routing_rule(desired.name)
+    current_value_does_not_exist! if config.nil?
+    ::Chef::Log.debug "Config is: #{config}"
+    description config['description']
+    mode config['mode']
+    matchers config['matchers'].sort
```

Comment:
the `.sort` call is not needed as it will be coerced.

Reply:
Done